jenkins-scripts/dsl/tools/get_collections_from_package_and_version.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+import sys
+import yaml
+
+
+# Function to find the collection name based on lib name and major version
+def find_collection(data, lib_name, major_version):
+  instances = []
+
+  for collection in data['collections']:
+    for lib in collection['libs']:
+      if lib['name'] == lib_name and lib['major_version'] == major_version:
+        instances.append(collection['name'])
+  return instances
+
+
+def get_major_version(version):
+  elements = version.split('.')
+  return int(elements[0])
+
+
+if len(sys.argv) < 3:
+  print(f"Usage: {sys.argv[0]} <lib_name> <major_version> <collection-yaml-file>")
+  sys.exit(1)
+
+lib_name = sys.argv[1]
+version = sys.argv[2]
+yaml_file = sys.argv[3]
+
+with open(yaml_file, 'r') as file:
+  data = yaml.safe_load(file)
+
+collection_names = find_collection(data, lib_name, get_major_version(version))
+print(f"{' '.join(collection_names)}")
```

Comment:
Consider putting this in an `if __main__` block so this module can be imported by other Python code in the future

Reply:
Implemented the __main__ and other improvements [3c62cde](https://github.com/gazebo-tooling/release-tools/pull/1148/commits/3c62cde6ba4103eb7ff4f73e6eb6335997a63ccf)